app/src/main/java/com/sunlightlabs/congress/services/ProPublica.java

```diff
@@ -0,0 +1,173 @@
+package com.sunlightlabs.congress.services;
+
+
+import android.util.Log;
+
+import com.sunlightlabs.android.congress.utils.HttpManager;
+import com.sunlightlabs.android.congress.utils.Utils;
+import com.sunlightlabs.congress.models.CongressException;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.UnsupportedEncodingException;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLEncoder;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+public class ProPublica {
+
+    // Pro Publica Per Page
+    public static int PPPP = 20;
```

Comment:
Not used, but this notes the hardcoded value of 20 items per page that Pro Publica documents.